File: cinema/angularclient/src/app/model/movie.ts
export class Movie {
    id: number;
    movieName: string;
    movieDescription: string;
    movieRunningTime: string;
    movieThumbnailUrl: string;

    
    constructor() {
        this.id = 0
        this.movieName = "";
        this.movieDescription ="";
        this.movieRunningTime = "";
        this.movieThumbnailUrl= "";
    }
}

File: cinema/src/main/java/com/example/cinema/movie/Movie.java
package com.example.cinema.movie;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Getter;
import lombok.Setter;

@Entity
@Setter
@Getter
public class Movie {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String movieName;
    private String movieDescription;
    private String movieRunningTime;
    private String movieThumbnailUrl;
}